Solaris specific patch to work around Studio compiler bug:
15779437 - SUNBT7154410 iropt runs out of space in large function
    with many calls to dlsym()

not suitable for upstream

--- a/glx/Makefile.am	2019-02-12 12:10:17.228282748 -0800
+++ b/glx/Makefile.am	2019-02-12 12:10:43.321585709 -0800
@@ -83,6 +83,9 @@
 
 libglx_la_LIBADD = $(DLOPEN_LIBS)
 
+# Workaround Studio 12.1 optimizer bug 15779437
+glapi_gentable.lo: CFLAGS = $(CCASFLAGS:-xO4=-xO0)
+
 libglxvnd_la_SOURCES = \
 	vndcmds.c \
 	vndext.c \
